server/controllers/session.controller.js

```diff
@@ -31,11 +31,9 @@ export function getSession(req, res) {
 }
 
 export function destroySession(req, res, next) {
-  req.logout((err) => {
-    if (err) {
-      next(err);
-      return;
-    }
-    res.json({ success: true });
+  res.cookie('sessionId', 'loggedout', {
+    expires: new Date(Date.now() + 10 * 1000),
+    httpOnly: true
```

Comment:
I'm trying to understand this.  It seems like you are setting a new cooking which will expire in 10 seconds, and that cookie has an invalid value.  Wouldn't it make more sense to use [`res.clearCookie`](http://expressjs.com/en/5x/api.html#res.clearCookie)?

Why delete the existing [`req.logout`](https://www.passportjs.org/concepts/authentication/logout/)?  If we do need to clear the cookie, it should be in addition to calling `req.logout`.

Reply:
OK, I'm gonna try it with a different method